polkadot-protocol/smart-contracts-basics/.pages

```diff
@@ -3,4 +3,5 @@ nav:
   - index.md
   - overview.md
   - polkavm-design.md
-  - evm-vs-polkavm.md
\ No newline at end of file
+  - evm-vs-polkavm.md
+  - networks.md
```

Comment:
Ahhh I just made a comment on the evm-vs-polkavm PR about these using the 'title': 'page.md' format. Can you either fix here or open a separate PR? Just for consistency across docs